main.py

```diff
@@ -70,6 +70,8 @@ def get_config():
                         help='Momentum factor for SGD only (default: %(default)s)')
     parser.add_argument('--patience', type=int, default=5,
                         help='Number of epochs to wait for improvement before early stopping (default: %(default)s)')
+    parser.add_argument('--normalize', type=bool, default=False,
```

Comment:
Use `action='store_true'` for flags.
https://docs.python.org/3/library/argparse.html#action

```suggestion
    parser.add_argument('--normalize', action='store_true',
```